### PR TITLE
dockerized ecc split

### PR DESCRIPTION
**Pull request by RobRossmiller-TG**

### CODE REVIEW

**Review by parkererickson-tg — APPROVED** [review]

**Comment by parkererickson-tg** [comment]
We also need to update the dev and main image build scripts to update the images as necessary in dockerhub

File: scripts/build_copilot.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# setup
+buildPath="build/copilot"
+cd ..
+rootPath=`pwd`
+rm -rf build
+mkdir build
+
+# copy assets into build
+cp -R copilot build
```
Comment (ahmed-tg) [inline scripts/build_copilot.sh:9]: should this be "build/copilot"

Comment (RobRossmiller-TG) [reply scripts/build_copilot.sh:9]: No, it get's copied into build as `build/copilot`. It's only different in the ECC to path get's renamed from `build/eventual-consistency-service` to `build/ecc`

Comment (RobRossmiller-TG) [reply scripts/build_copilot.sh:9]: Or do you mean stylistically, it should be `build/copilot`?

Comment (ahmed-tg) [reply scripts/build_copilot.sh:9]: oh got it thanks!

File: docker-compose-with-apps.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      LLM_CONFIG: "/code/configs/llm_config.json"
+      DB_CONFIG: "/code/configs/db_config.json"
+      MILVUS_CONFIG: "/code/configs/milvus_config.json"
+      LOGLEVEL: "INFO"
+      MILVUS_HOST: "milvus-standalone"
+    volumes:
+      - ./configs/:/code/configs
+      - ./common:/code/common
+
+  etcd:
```
Comment (ahmed-tg) [inline docker-compose-with-apps.yml:39]: good idea separating this out

**Pull request merged**